### PR TITLE
chore: updating QA env to 2020.12

### PR DESCRIPTION
**Pull request by atharvar28**
Updating PRC_QA to 2020.12 release version

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-covid19.planx-pla.net/manifest.json
## :warning: Services on branch
- auspice

File: qa-covid19.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -4,30 +4,31 @@
...
+    "sheepdog": "quay.io/cdis/sheepdog:2020.12",
     "portal": "quay.io/cdis/data-portal:2.40.0",
-    "tube": "quay.io/cdis/tube:2020.11",
+    "tube": "quay.io/cdis/tube:2020.12",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "spark": "quay.io/cdis/gen3-spark:2020.11",
-    "hatchery": "quay.io/cdis/hatchery:2020.11",
+    "spark": "quay.io/cdis/gen3-spark:2020.12",
+    "hatchery": "quay.io/cdis/hatchery:2020.12",
     "mariner": "quay.io/cdis/mariner-server:1.1.1",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:22]: the 3 mariner versions can be set to "2020.12"

File: qa-covid19.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -4,30 +4,31 @@
...
-    "wts": "quay.io/cdis/workspace-token-service:2020.11",
-    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:1.0.4",
+    "wts": "quay.io/cdis/workspace-token-service:2020.12",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.12",
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/manifest.json:24]: 1.0.4 contains newer commits.

**Review by paulineribeyre — CHANGES_REQUESTED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# qa-covid19.planx-pla.net/manifest.json
## :warning: Services on branch
- auspice

**Pull request merged**